packages/supabase_flutter/example/pubspec.yaml

```diff
@@ -21,3 +21,21 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
+melos_managed_dependency_overrides: functions_client,gotrue,postgrest,realtime_client,storage_client,yet_another_json_isolate
```

Comment:
I'm unsure about these changes here. Did you run any command to produce these lines? Is there no way to include the example in the bootstrap to avoid modifying the overrides in the pubspec.yaml?

Reply:
What is the concern? For now, this solves our issue, ensuring that the package works on Flutter web. I can see that this can cause slight inconvenience for anyone viewing the example directly, but that's not the top priority issue that we have right now, and we can take our time to figure out if there is a better way of overriding the dependencies.

Reply:
Okay, I have updated the melos config instead of the pubspec.yaml file of the example.

Reply:
And of course now the tests are failing 😂

Reply:
I wouldn't add the packages to the bootstrap section. Just the oneline under packages

Reply:
The CI is probably failing, because pub get for the example app is running before some of the other packages. I have to run now for a bit, but will take a look when I come back.

Reply:
The issue was, that melso tried to use the flutter cli, because the example is a flutter package and was not excluded like the supabase_flutter package everywhere manually.